packages/cli/README.md

```diff
@@ -15,49 +15,109 @@
+
+    Below command installs completion script in `~/.bashrc`.<br /><br />
+
+    ```bash
+    si completion install
+    ```
+
+    Running command `si completion bash` prints script to the terminal.
+
+## Commands <!-- omit in toc -->
 
-Once installed, the CLI is available as the command `si`:
+- [Show help](#show-help)
+- [Manage config](#manage-config)
+- [Hub operations](#hub-operations)
+- [Create a package](#create-a-package)
+- [Sequence operations](#sequence-operations)
+- [Instance operations](#instance-operations)
 
-```md
-Usage: si [options...] | si [command] [options...]
+## Show help
 
-General options
+Check the available commands by typing `si --help` in the terminal.
 
-* `-L, --log-level <level>  Specify log level (default: "trace")`
-* `-a, --api-url <url>      Specify base API url (default: "http://localhost:8000/api/v1")`
-* `-f, --format <value>     Specify display formatting: json or pretty (default: "pretty")`
-* `-h, --help               display help for command`
+```bash
+USAGE
+   si [options...]
+   si [command] [options...]
+   si [command] [subcommand] [options...]
+
+GLOBAL OPTIONS
+   -L, --log-level <level>     specify log level (default: "trace")
+   -f, --format <value>        specify display formatting: json or pretty (default: "pretty")
+   -h, --help                  display help for command
+   -v, --version               display version
+
+COMMANDS
+   config, c                   contains default configuration settings that are
+   hub                         allows to run programs in different data centers, computers or devices in local network
+   sequence, seq               operations on sequence of chained functions aka program
+   instance, inst              operations on running sequence aka computing instance
+   topic                       publish/subscribe operations allows to manage data flow
+   template, tmpl, init        create template and start working on your sequence
+```
 
-Commands
+Show subcommand help by providing `--help` or `-h` option after each as in example below.
 
-* `pack [options]`
-* `host [command]           something`
-* `config, c [command]       configuration file operations`
-* `sequence, seq [command]   operations on sequence`
-* `instance, inst [command]  operations on instance`
-* `help [command]           display help for command`
+```bash
+USAGE
+    si [command] --help
 
-Show sequence and instance help by providing --help option after each.
+EXAMPLE
+    si sequence -h
 ```
 
-## Set up config
+## Manage config
+
+1. First set environmental vale.<br />
+   In order to use STH CLI commands the **environment should be set to develop**. If this is the first installation of Scramjet CLI, the **default value** of the environment **is set to `develop` mode**. To check the config values use `si config print` command.<br /><br />
 
```

Comment:
It's a good idea to first ask the user to make sure they have the correct environment value set up in the config.

I'm proposing here a slight change of naming, which would be aligned with the CLI. I checked that CLI uses only "environment" ("env") key name, not "mode" (as I used to call it). If we stay with the proposed "environment", maybe we could write it like that? I added a couple of sentences and incorporated in them yours. Let me know what you think:

1. Please check that the CLI environment is set to development.<br />

The CLI interface can work in either the development or production environment. 

The CLI development environment communicates with the Transform Hub. If this is your first installation of Scramjet CLI, the environment should already be set development since it is the **default value** of the CLI environment.

The CLI production environment enables the user to deploy their programs to the [beta platform](https://scramjet.org/#join-beta).

If you wish you run your programs on your machines, please make sure the environment is set to development.  To check the config values use `si config print` command.<br /><br />

Reply:
Maybe we could link here also to your nice section # Cloud Platform Beta is coming